eventlet/patcher.py

```diff
@@ -1,5 +1,7 @@
 import imp
+import os
```

Comment:
I don't think we can be sure that at the time of accessing this `os` it's not patched already (multiple `monkey_patch` calls are probably legal if they patch separate things?), correct me if I'm wrong

Reply:
Ya, but `register_at_fork` is not patched, so either system or patched `os` of same value for this particular change.

Reply:
Fair enough. I'd import `register_at_fork` only from the module to not leave the whole maybe-patched-maybe-not `os` around for someone to use accidentally.

Reply:
done